lib/spack/spack/binary_distribution.py

```diff
@@ -377,9 +389,15 @@ def make_package_relative(workdir, prefix, allow_root):
         cur_path_names.append(os.path.join(workdir, filename))
     relocate.make_binary_relative(cur_path_names, orig_path_names,
                                   old_path, allow_root)
+    orig_path_names = list()
+    cur_path_names = list()
```

Comment:
This is minor and bikesheddy but I really really prefer `[]` for this.  I use `list()` when I want to create a `list` from a sequence but `[]` for new empty list.

Reply:
I have the same preference, but I kept it this way to mirror what we already have elsewhere in this function. LMK if you prefer I change it.